src/components/menu/examples/menu-basic-secondary-text.tsx

```diff
@@ -0,0 +1,66 @@
+import { MenuItem, ListSeparator } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * With `secondaryText`
+ *
+ * Menu items can display secondary text as well. By default, the secondary text
+ * will be displayed in two lines, and then get truncated.
+ *
+ * :::important
+ * Keep in mind that menu lists stretch as much as they can to display
+ * their content. You must therefore specify a maximum with for the menus with
+ * long secondary text, otherwise they will fill out the entire screen width.
+ *
+ * But do not forget that menus should behave responsively, thus using a fixed `width`
+ * should be avoided. To make the width responsive, try using the `min()` function.
+ * This function selects the smallest value from a list of comma-separated expressions
+ * which are placed within the parentheses.
+ *
+ * For example, `--menu-surface-width: min(90vw, 40rem);` will output
+ * `width: min(90wv, 40rem);` which will tell the browser to render the menu
+ * content in a 40rem-wide grid, as long as 90% of the viewport's width (`90vw`)
+ * is larger than `40rem`.
```

Comment:
That sentence is a bit clunky. I think the below is a little better, but I'm not sure. What do you think?

```suggestion
 * content in a grid that's allowed to take up 90% of the viewport's width (`90vw`)
 * up to a maximum of `40rem`.
```